### PR TITLE
Allow remote use of the autoclave

### PR DESCRIPTION
**Pull request by NEONZARIN**
#### Summary

```SUMMARY: Balance "Allow remote use of the autoclave"```

#### Purpose of change

This tool is TOO heavy and bulky to take it in your hands each time to activate / recharge. It would be easier to put it in a container and activate without picking it up every time.

#### Describe the solution

Added "ALLOWS_REMOTE_USE" tag..

#### Describe alternatives you've considered

N/A

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
Please follow the template when you fill out a PR. To get it back, click the `New Pull Request` button under the pulls page, and copy the text there over to this.

**Comment by Fris0uman** [comment]
What is it suppose to do? I don't think allowremote_use will do anything with somerhing that's not used in any recipe.

**Comment by NEONZARIN** [comment]
This tool is TOO heavy and bulky to take it in your hands each time to activate / recharge. Wouldn't it be easier to put it in a container and activate without taking his hands every time?

Excessive battery consumption does not allow to complete the sterilization process.

**Comment by Fris0uman** [comment]
Alright, remote use is a pretty good idea then ^^

File: data/json/items/tools.json [hunk 1]
```diff
@@ -5547,7 +5547,8 @@
     "symbol": "A",
     "color": "yellow",
     "use_action": "AUTOCLAVE",
-    "power_draw": 1500000,
+	"flags": [ "ALLOWS_REMOTE_USE" ],
```
Comment (AMurkin) [inline data/json/items/tools.json]:
```suggestion
    "flags": [ "ALLOWS_REMOTE_USE" ],
```

**Comment by Zireael07** [comment]
Reduced power draw is a serious balance change...

**Comment by NEONZARIN** [comment]
image
image
image
Yes, of course, but is such a battery requirement normal?  I think this is too much.

**Comment by Zireael07** [comment]
I think the requirement was intended, but we're missing even bigger batteries - same deal as with electric jackhammers.

**Pull request merged**